_tests/travis-checks

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+# Execute Markdown Linter
+mdl README.md _posts _pages --style _linter/markdown-linter-style.rb
+
+# Github Pages needs the baseurl field to be "m-lab.github.io" to render properly in forked 
+# repos, but when we deploy to production, we need the baseurl field to be empty. This 
+# command replaces "m-lab.github.io" with an empty string.
+sed -e '/^baseurl/s/\/m-lab\.github\.io//' _config.yml > _config-deploy.yml
+
+# sets --disable-external flag for HTMLProofer if quick argument exists
```

Comment:
Right, good call.

Fixed in https://github.com/m-lab/m-lab.github.io/pull/81/commits/eabd7618abc1915efa0d31b50e1c9b29e56e6078